styles/scss/core/_variables.scss

```diff
@@ -468,6 +468,7 @@ $border-radius-sm: var(--pgn-size-border-radius-sm) !default;
 $rounded-pill: var(--pgn-size-rounded-pill) !default;
 
 $level-1-box-shadow: var(--pgn-elevation-box-shadow-level-1) !default;
+
```

Comment:
nit: unnecessary new line (might make sense to keep `$level-1-box-shadow` co-located with the other `$level-*` SCSS variables below.

Reply:
Agree, removed in [`0905115` (#3222)](https://github.com/openedx/paragon/pull/3222/commits/09051155b1aa93b654844015ef9289b0387736d2#diff-047cf3e37a3d52f4ec293da65abc4a20d5364ed9effd900f64b33c7981110976L471)